Samples/src/ReversibleLogicSynthesis/ReversibleLogicSynthesis.qs

```diff
@@ -93,18 +93,18 @@ namespace Microsoft.Quantum.Samples.ReversibleLogicSynthesis {
     /// Sequence(23, 29); // [23, 24, 25, 26, 27, 28, 29]
     /// ```
     function Sequence (from : Int, to : Int) : Int[] {
```

Comment:
We should probably have a good initialization function in the Arrays module...
On one hand we need "conversions" between arrays and ranges (and have a conversation on related type system topics), on the other hand we should also have array initializers taking initialization functions.

Reply:
Very much agreed, let's open an issue for that on QuantumLibraries, then.